man/ocaml.1

```diff
@@ -361,7 +360,7 @@ and look up its capabilities in the terminal database.
 .B .ocamlinit
 lookup procedure (see above).
 .SH SEE ALSO
-.BR ocamlc (1), \ ocamlopt (1), \ ocamlrun (1).
+.BR ocamlc "(1), " ocamlopt "(1), " ocamlrun " (1)."
```

Comment:
There is a  superfluous space after ocamlrun, but otherwise I don't see any effect for this change. Is it here to keep a homogeneous format?